net/shorewall-lite/patches/120-logfile.patch

```diff
@@ -0,0 +1,13 @@
+Index: shorewall-lite-5.1.7.2/shorewall-lite.conf
+===================================================================
+--- shorewall-lite-5.1.7.2.orig/shorewall-lite.conf	2017-10-11 08:19:35.565847520 +0200
++++ shorewall-lite-5.1.7.2/shorewall-lite.conf	2017-10-11 08:19:35.561847733 +0200
+@@ -28,7 +28,7 @@
+ #			       L O G G I N G
+ ###############################################################################
+ 
+-LOGFILE=
++LOGFILE=/dev/null
```

Comment:
Why is logging discarded?  Why not use the syslog service?

Reply:
Default Shorewall logs to syslog and can be read with Logread. The logfile parameter in the configfile points to a separate file which I have disabled by default to spare diskspace.

Reply:
Okay.